### PR TITLE
fix: update demo

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: update demo |
| 🇨🇳 Chinese | fix: update demo |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/qrcode/interface.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ interface ImageSettings {
 interface QRProps {
   value: string;
   size?: number;
-  level?: string;
```
Comment (li-jia-nan) [inline components/qrcode/interface.ts:15]: 还发现了一个多余的属性，删掉了

Comment (yoyo837) [reply components/qrcode/interface.ts:15]: image

Comment (yoyo837) [reply components/qrcode/interface.ts:15]: > 还发现了一个多余的属性，删掉了

这不是在用么?

File: components/qrcode/index.tsx [hunk 2]
```diff
@@ -59,6 +59,14 @@ const QRCode: React.FC<QRCodeProps> = (props) => {
     return null;
   }
 
+  if (process.env.NODE_ENV !== 'production') {
+    warning(
+      !(icon && errorLevel === 'L'),
+      'QRCode',
+      'errorLevel `L` is not recommended to be used with `icon`, for low errorLevel will affect scanning result',
```
Comment (MadCcc) [inline components/qrcode/index.tsx:66]:
```suggestion
      'ErrorLevel `L` is not recommended to be used with `icon`, for scanning result would be affected by low level.',
```

**Comment by afc163** [comment]
ci 挂了

**Comment by li-jia-nan** [comment]
> ci 挂了

在看，我家里电脑梯子坏了，代码推不上去，可能要明天去公司看了

**Review by MadCcc — APPROVED** [review]

**Pull request merged**